ambry-router/src/main/java/com/github/ambry/router/GetBlobOperation.java

```diff
@@ -840,11 +840,6 @@ void handleBody(InputStream payload, MessageMetadata messageMetadata, MessageInf
       // from concurrent operations.
       if (!successfullyDeserialized) {
         BlobData blobData = MessageFormatRecord.deserializeBlob(payload);
-        // Note that for segment GET there is only one chunk to get, so we can be sure that its this one.
```

Comment:
Removing this because of three reasons
1. This method only deal with data chunk, and changing blob size in property is already done in metadata chunk.
2. There is a race condition when the frontend is writing header back to client but the blob size is changed at the same time.
3. Here the size is undecrypted size, we have to run decryption to get the real size. 
@ankagrawal